app/views/layouts/application.mobile.haml

```diff
@@ -8,7 +8,7 @@
     %title
       = pod_name
 
-    %meta{:name => "description", :content => "diaspora* mobile"}/
+    %meta{:name => "description", :content => "#{pod_name} mobile"}/
```

Comment:
For the desktop view we do the following: 

https://github.com/diaspora/diaspora/blob/b319e0caa24c7387d1040668284bec747da08f9f/app/views/layouts/application.html.haml#L13
https://github.com/diaspora/diaspora/blob/b319e0caa24c7387d1040668284bec747da08f9f/app/helpers/meta_data_helper.rb#L29-L38

This uses [the description from the diaspora.yml file](https://github.com/diaspora/diaspora/blob/b319e0caa24c7387d1040668284bec747da08f9f/config/diaspora.yml.example#L557) which should be appropriate for both desktop and mobile views so I'd prefer some unification here.